airbyte-integrations/connectors/source-posthog/metadata.yaml

```diff
@@ -23,6 +23,13 @@ data:
       enabled: true
   releaseStage: beta
   supportLevel: community
+  releases:
+    breakingChanges:
+      1.0.0:
+        message:
+          The `event` field in the `events` stream has been corrected to the proper data type.
+          It was previously set as `[object, string]`, but the correct type is only `string`.
```

Comment:
The `event` field in the `events` stream has been corrected to the proper data type. To apply this change, refresh the schema for the 'events' stream and reset your data. For more information visit <link to migration guide>